IBAnimatable/FillDesignable.swift

```diff
@@ -34,6 +37,62 @@ public extension FillDesignable where Self: UITableViewCell {
     if let unwrappedFillColor = fillColor {
       backgroundColor = unwrappedFillColor
       contentView.backgroundColor = unwrappedFillColor
+    } else if let unwrappedPredefinedColor = predefinedColorFromString(predefinedColor) {
+      backgroundColor = unwrappedPredefinedColor
+      contentView.backgroundColor = unwrappedPredefinedColor
     }
   }
 }
+
+private extension FillDesignable {
+  
+  func predefinedColorFromString(predefinedColor: String?) -> UIColor? {
+    guard let unwrappedColorTypeString = predefinedColor, colorType = ColorType(rawValue: unwrappedColorTypeString) else {
+      return nil
+    }
+    
+    switch colorType {
+    case .Emerland:
```

Comment:
Do you think we should prefix them as FlatEmerland to support MaterialRed later on ?